lib/src/migrators/module/member_declaration.dart

```diff
@@ -77,3 +79,9 @@ class MemberDeclaration<T extends SassNode> {
 
   int get hashCode => member.hashCode;
 }
+
+/// Returns true if [forward] is non-null and inside an import-only file.
+bool _isImportOnly(ForwardRule forward) =>
```

Comment:
Nit: does this need to take `forward` as input?

Reply:
Good point. Changed this to just take a URL and moved it to `utils.dart`, so it can also be used by the references visitor.